static/app/utils/replays/hydrateBreadcrumbs.tsx

```diff
@@ -18,6 +36,11 @@ export default function hydrateBreadcrumbs(
         const time = new Date(frame.timestamp * 1000);
         invariant(isValidDate(time), 'breadcrumbFrame.timestamp is invalid');
 
+        if (frame.category === 'replay.hydrate') {
+          frame.data = {
+            mutations: findCloseMutations(time, rrwebFrames),
```

Comment:
@scttcper If you're iterating on the hydration errors this is a good spot (And in the find* method) to mess around.